dev_scripts/qa.py

```diff
@@ -3,14 +3,21 @@
 import abc
 import argparse
 import difflib
+import json
 import logging
 import re
 import selectors
 import subprocess
 import sys
+import urllib.request
+from pathlib import Path
 
 logger = logging.getLogger(__name__)
 
+PYTHON_VERSION_STR = "3.12"
+PYTHON_VERSION = [int(num) for num in PYTHON_VERSION_STR.split(".")]
```

Comment:
(nitpick, feel free to ignore if you don't feel it's worth it): 

I think we could have this done as a function rather than a constant, because it's actually converting something.

Reply:
You're right, I'll factor this logic out.

Reply:
Actually, `PYTHON_VERSION` was not used anywhere :grimacing:. So, I just removed it and renamed the `PYTHON_VERSION_STR` variable.